geometry_msgs/Point first_point
geometry_msgs/Point last_point

